src/app_model/registries/_keybindings_reg.py

```diff
@@ -26,12 +26,30 @@ class _RegisteredKeyBinding(NamedTuple):
 
 
 class KeyBindingsRegistry:
-    """Registry for keybindings."""
+    """Registry for keybindings.
+
+    Attributes
+    ----------
+    filter_keybinding : Callable[[KeyBinding], str] | None
+        Function for filtering out invalid `KeyBinding`s.
+        Callable should return an error message string if `KeyBinding` is invalid
+        or empty string if `KeyBinding` is valid.
```

Comment:
```suggestion
        Optional function for applying additional `KeyBinding` validation.
        Callable should accept a `KeyBinding` object and return an error message (`str`)
        if `KeyBinding` is rejected, or empty string otherwise.
```

trying to make it clearer that the KeyBinding object itself will already be valid, and that this is a app-specific "preference"